stable

```diff
@@ -1 +1 @@
-2.2
\ No newline at end of file
+2.3.0
```

Comment:
For consistency with previous releases we should create a `2.3` symlink to `2.3.0`, and let this `stable` symlink point to `2.3`.